scripts/obtain/install.sh

```diff
@@ -4,8 +4,25 @@
 
 # Note: This script should be compatible with the dash shell used in Ubuntu. So avoid bashisms! See https://wiki.ubuntu.com/DashAsBinSh for more info
 
-#setup some colors to use. These need to work in fairly limited shells, like the Ubuntu Docker container where there are only 8 colors.
-#See if stdout is a terminal
+# Stop script on NZEC
```

Comment:
Assuming you're willing to not use bashims, that would be better.  FWIW, CoreCLR and CoreFX and WCF all require bash to build, but maybe taking some pain to have one less dependency to install is worth it.